static/js/util/withTracker.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+/* global SETTINGS: false */
+
+// From https://github.com/ReactTraining/react-router/issues/4278#issuecomment-299692502
+import React from 'react';
+import ga from 'react-ga';
+
+const withTracker = (WrappedComponent: Class<React.Component<*, *, *>>) => {
+  const debug = SETTINGS.reactGaDebug === "true";
+
+  if (SETTINGS.gaTrackingID) {
+    ga.initialize(SETTINGS.gaTrackingID, { debug: debug });
+  }
+
+  const trackPage = (page: string) => {
+    ga.pageview(page);
+  };
```

Comment:
is this function necessary for some reason? why not just call `ga.pageview` below, on line 21?